sdk/cosmos/azure-cosmos/azure/cosmos/partition_key.py

```diff
@@ -86,3 +141,165 @@ def version(self):
     @version.setter
     def version(self, value):
         self["version"] = value
+
+    def get_epk_range_for_prefix_partition_key(self, pk_value: list) -> Range:
+        if self.kind != "MultiHash":
+            raise ValueError("Unsupported partition definition kind for partial key operations")
+
+        if len(pk_value) >= len(self["paths"]):
+            raise ValueError("Too many partition key components")
+
+        min_epk = self.get_effective_partition_key_string(pk_value)
+        if min_epk == MinimumInclusiveEffectivePartitionKey:
+            min_epk = str(MinimumInclusiveEffectivePartitionKey)
+            return Range(min_epk, min_epk, True, False)
```

Comment:
In .NET when getting the Range we use (min, max, true, true). Please double check to make sure it is different here. 
[ref](https://github.com/Azure/azure-cosmos-dotnet-v3/blob/be0c0981b80c2ba18b73588520a80e9437b289de/Microsoft.Azure.Cosmos/src/FeedRange/FeedRanges/FeedRangePartitionKey.cs#L34C17-L43C20)

Reply:
In the case of Prefix Partitions it should be (min, max, True, False).